bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -544,10 +544,10 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm60_params.c241017.nc</paramfile>
-<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c241017.nc</paramfile>
-<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c241017.nc</paramfile>
-<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c241017.nc</paramfile>
+<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm60_params.c241119.nc</paramfile>
+<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm60_params.c241119.nc</paramfile>
+<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c241119.nc</paramfile>
+<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c241119.nc</paramfile>
```

Comment:
Reminder to 
- [ ] ./rimport the new paramfile when we're ready to merge this

Reply:
Tests on derecho
PASS ./build-namelist_test.pl

Tests on derecho and izumi
IN PROGRESS `./run_sys_tests -s aux_clm -c ctsm5.3.013 --skip-generate`

Reply:
On izumi this command shows that aux_clm was OK:
./cs.status.fails | grep -v NLCOMP | grep -v PASS | grep -v 'EXPECTED FAILURE' | grep -v 'me baseline files were mis'

Reply:
On derecho this command shows a few failures that I need to resolve:
./cs.status.fails | grep -v NLCOMP | grep -v PASS | grep -v 'EXPECTED FAILURE'
```
    FAIL ERP_D_Ld5.ne30pg3_t232.IHistClm51Sp.derecho_intel.clm-default BASELINE ctsm5.3.013: DIFF
    FAIL ERP_D_Ld9.ne30pg3_t232.I1850Clm51BgcCropG.derecho_intel.clm-clm51cam6LndTuningMode BASELINE ctsm5.3.013: DIFF
    FAIL ERS_Ld3.f10_f10_mg37.I2000Clm60Bgc.derecho_intel.clm-ciso_cwd_hr RUN time=42
    FAIL ERS_Ld3.f10_f10_mg37.I2000Clm60Bgc.derecho_intel.clm-ciso_cwd_hr--clm-matrixcnOn RUN time=39
    FAIL SMS_Lm1.f10_f10_mg37.I1850Clm60Bgc.derecho_intel.clm-clm60_monthly_matrixcn_soilCN30 RUN time=23
```

Reply:
The last 3 are expected [as per Erik's post](https://github.com/ESCOMP/CTSM/pull/2869#issuecomment-2487759533).

Reply:
The two DIFF failures show this diff in lnd_in
```
<  paramfile = '/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/paramdata/ctsm60_params.c241119.nc'
---
>  paramfile = '/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/paramdata/ctsm51_params.c241017.nc'
```
and these paramfiles differ more than just due to the newly introduced params. Also the second of the two DIFF failures has a Clm60 counterpart that passes and clm51 tests go away in ctsm5.3.014:
`ERP_D_Ld9.ne30pg3_t232.I1850Clm60BgcCropG.derecho_intel.clm-clm60cam7LndTuningModeLDust`

So I consider aux_clm on derecho OK.

Reply:
@ekluzek
`./rimport -file lnd/clm2/paramdata/clm50_params.c241119.nc`
returns this error
`/glade/p/cesmdata/cseg/cseg/inputdata/lnd/clm2/paramdata/clm50_params.c241119.nc does not exist`
Can you tell whether it's due to file permissions or a different problem that I should report to somebody?

Reply:
./rimport will have to wait for when the tool/repository works again. Jim Edwards filed a ticket with help@ucar.edu.